src/main/java/mate/academy/service/UserService.java

```diff
@@ -3,7 +3,7 @@
 import mate.academy.model.User;
 
 public class UserService {
-    private static final User[] users = new User[] {
+    public static final User[] users = new User[] {
```

Comment:
the field must be the private, revert it
```suggestion
    private static final User[] users = new User[] {
```